src/main/webapp/components/sources-info/sources-info.js

```diff
@@ -63,14 +64,13 @@ export const SourcesInfo = props => {
           <List>
             {sources.map(source => {
               const Icon = getIcon(source)
-
               return (
-                <ListItem key={source.sourceId}>
+                <ListItem key={source.sourceId || source.value}>
```

Comment:
:pencil2: Can we normalize the props so we only need one of these fields? (i.e. give base sources the `value` property or give facet sources the `sourceId` property before passing them into this component)